transformation/src/jade-schema/tables/experiment_activity.table.json

```diff
@@ -89,11 +89,6 @@
       "datatype": "string",
       "type": "required"
     },
-    {
-      "name": "status",
-      "datatype": "string",
-      "type": "required"
-    },
```

Comment:
Dan made a good point that the status field should have been removed from all of the tables last week (I had only removed it from the files). This change is a bit tangential to the rest of this PR, but I am including it here so that it gets merged before other Encode transformation tickets are started.